package part2.ch5.ex13.page190;

import java.io.IOException;

class Program {
	public static void main(String[] args) throws IOException {

		int test1;
		test1 = System.in.read();
		System.out.printf("test1의 값은 %d\n", test1);
		test1 = System.in.read();
		System.out.printf("test1의 값은 %d\n", test1);
		test1 = System.in.read();
		System.out.printf("test1의 값은 %d\n", test1);
		test1 = System.in.read();
		System.out.printf("test1의 값은 %d\n", test1);
		
	}
}